.golangci.yml

```diff
@@ -78,6 +78,7 @@ issues:
     - "Error return value of `cmd.MarkFlag.+` is not checked"
     - "Error return value of `net.SplitHostPort` is not checked"
     - "Error return value of `w.Write` is not checked"
+    - "Function 'handle' has too many statements"
```

Comment:
We keep silence these linters or increase limits. Maybe we should disable it for good.

Reply:
OK.